README.md

```diff
@@ -1,3 +1,102 @@
-TuxedoJS
-========
+# TuxedoJS
+
 ![Tux Logo](Tux-full-logo.png "Tux-Logo")
+
+
+> A Front-End Javascript Framework built on Facebook’s powerful React view layer and Flux architecture.
+
+React and Flux are two novel and exciting approaches to front-end development. TuxedoJS implements a more accessible and semantic toolkit that leverages these technologies, abstracting away complexity and extending core functionality. TuxedoJS comes from Tux - a merging of React and Flux.
+
+## Table of Contents
+
+1. [Usage](#usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+    1. [Installing Dependencies](#installing-dependencies)
+    1. [Tasks](#tasks)
+    1. [Roadmap](#roadmap)
+1. [Graceful Degradation](#graceful-degradation)
+1. [Credits and Tech Stack](#credits-and-tech-stack)
+1. [Contributing](#interested-in-contributing)
+1. [Team](#team)
+
+## Usage
+
+TuxedoJS is a modular framework that provides tools for easily rendering components, client-side routing, animating transitions, validating inputs, and managing app data.  Install it via a package manager and include the modules you need to quickly start building your app.
+
+See the full doc site [here](http://www.tuxedojs.org).
+
+## Requirements
+
+- Node 0.10.x
+
+## Development
+
+View the entire [Getting Started]() tutorial to begin hacking with Tux.
+
+### Installing Dependencies
+
+After cloning this repo,  within the root directory:
+
+    npm install tux
+
+### Tasks
+
+Tux is built off of CommonJS and uses a combination of Browserify and Reactify to build client-side dependencies and automatically compile JSX syntax. Because of this, we highly recommend installing the following modules to aid in development while using Tux:
```

Comment:
Change to: Tux is built with CommonJS and thus you will need a compiler such as `Browserify` or `webpack`. In our case, we use `Browserify` for compiling, `Reactify` for compiling JSX, `Envify` for accessing NODE_ENV variables in the browser (great for automatically turning dev tools on and off), and `Watchify` for automatic compiling.